code/modules/reagents/reagent_containers/food/snacks/dough.dm

```diff
@@ -0,0 +1,145 @@
+/obj/item/chems/food/snacks/dough
+	name = "dough"
+	desc = "A piece of dough."
+	icon = 'icons/obj/food_ingredients.dmi'
+	icon_state = "dough"
+	bitesize = 2
+	center_of_mass = @"{'x':16,'y':13}"
+	nutriment_desc = list("dough" = 3)
+	nutriment_amt = 3
+	nutriment_type = /decl/material/liquid/nutriment/bread
+
+// Dough + rolling pin = flat dough
+/obj/item/chems/food/snacks/dough/attackby(obj/item/W, mob/user)
+	if(istype(W,/obj/item/kitchen/rollingpin))
+		new /obj/item/chems/food/snacks/sliceable/flatdough(src)
```

Comment:
Should maybe make all of these spawn in `get_turf`, or use `dropInto`. It does work due to a `/snacks/Destroy` override, but that override is also kind of undesirable. `loc` also doesn't really work, as that's probably the mob.